src/main/features/claimant-response/views/confirmation.njk

```diff
@@ -41,7 +41,10 @@
           <span class="bold">{{ confirmationDate | date }}</span>
         </p>
         {% if directionsQuestionnaireEnabled %}
-        <p><a href="http://hmcts-cmc-prototype.herokuapp.com/prototypes/features-to-be-built/claimant/correspondence/dqs">Download your hearing requirements</a> (PDF, 123KB)</p>
+        <p class="receipt-download-container">
+          {{ internalLink(t('Download your hearing requirements'), ClaimantResponsePaths.claimantReceiptReceiver.evaluateUri({ externalId: claim.externalId })) }}
+          {{ t('(PDF)') }}
```

Comment:
Not sure if we have test in other places where we download pdf! if we do, it will be good to add test here too...

Reply:
yeah, I have added, pls re-review. @KiranVarma02